.env.sample

```
@@ -6,3 +6,5 @@ AWS_SECRET_ACCESS_KEY=
 S3_BUCKET_NAME=
 AWS_BUCKET_REGION=
 DOCKER_ENABLED=false
+NEW_RELIC_API_KEY=api_key
+NEW_RELIC_APP_NAME='Rails API base - development'
```

```suggestion
NEW_RELIC_APP_NAME='rails_api_base - development'
```

so `bootstrap.sh` replaces it